datapackage.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "co2-fossil-by-nation",
-  "title": "CO2 Emissions from Fossil Fuels since 1751, By Nation",
+  "title": "National CO2 emissions from fossil fuels and cement manufacture",
```

Comment:
@gradedSystem I'm not sure if we should update the title. But I wanted to confirm if this new source actually includes co2 emission data from cement manufacture?

Reply:
@anuveyatsu It includes the data from cement manufacture